app/shared.css

```diff
@@ -151,6 +151,10 @@ Let us know if you think you've found one we'd like. 😏
 	padding: 0;
 }
 
+.page-grid-footer-link.active {
+	text-decoration: underline;
```

Comment:
This works! I do want to think a bit more holistically on the design system (we're starting to have a bunch of different treatments for emphasis)... but I like this a lot for now. 😄 

cc @tjwds - thoughts?

Reply:
Agreed on the holistic design system comment and that this work for now 👌

Reply:
Yep, agreed, thanks!  I filed #69.

Reply:
Whoo!